wsdlgo/encoder.go

```diff
@@ -1023,7 +1066,10 @@ func (ge *goEncoder) genElementField(w io.Writer, el *wsdl.Element) {
 		}
 	}
 	tag := el.Name
-	fmt.Fprintf(w, "%s ", strings.Title(el.Name))
+
+	// Titlecase the name, then fix Go lint warning when using "Id" instead of "ID"
+	name := regexp.MustCompile("(.*)Id$").ReplaceAllString(strings.Title(el.Name), "${1}ID")
```

Comment:
I'm not following your ditto--but I wound up dropping this into a `scrubName` method and updated names that should be scrubbed.

At some point, scrubbing should apply to naming for complex types as well--but that's outside the scope of what I need this for.